Content.Server/Chemistry/Metabolism/OverdoseMetabolism.cs

```diff
@@ -0,0 +1,63 @@
+using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Content.Shared.Damage;
+using Content.Shared.Chemistry.Metabolizable;
+using Content.Shared.Chemistry.Reagent;
+using Content.Shared.Damage.Components;
+using Content.Server.Chemistry.Components;
+
+namespace Content.Server.Chemistry.Metabolism
+{
+    /// <summary>
+    /// Deals a certain amount of damage once a certain amount of reagent is ingested at once.
```

Comment:
this being a 'metabolism' seems really weird since an 'overdose' isn't something that metabolizes anything; this really should be made into a more general system for overdoses, so you can have different overdose effects; ideally you'd specify this on the reagent the same way that metabolisms are specified

Reply:
Yeah it needs specifying per species as well. I forget how we were going to handle this with metabolism.

Also the fact that we need overdoseReagent when it's already stored on the reagent itself in this PR is gigasus regardless.